website/src/components/views/sequencingEfforts/GenericSequencingEffortsPage.astro

```diff
@@ -42,7 +42,16 @@ const lineageFilterConfigs = getLineageFilterConfigs(
 );
 ---
 
-<SingleVariantOrganismPageLayout view={view}>
+<SingleVariantOrganismPageLayout
+    view={view}
+    downloadLinks={[
+        {
+            label: 'Download accessions',
+            filter: datasetLapisFilter,
```

Comment:
```suggestion
            filter: variantLapisFilter,
```
(to fix the error)
But I would prefer renaming the variable to "lapisFilter", since we don't show variants on the sequencing efforst pages.